packages/application-shell/src/components/navbar/menu-items.tsx

```diff
@@ -249,6 +252,8 @@ type MenuItemProps = {
     | FocusEventHandler<HTMLElement>;
   children: ReactNode;
   identifier?: string;
+  handleSubMenuKeyDown?: (e: React.KeyboardEvent<HTMLLIElement>) => void;
```

Comment:
This seems this prop is never used

Reply:
Updated here: https://github.com/commercetools/merchant-center-application-kit/pull/3482/commits/4e07032305fa1fd7a4aa2a4655e77c65b2a8ad5c
Thank you